README.md

```diff
@@ -98,31 +98,30 @@ rails s
 
 - [ ] **More methods**
 
-<p align="right">([back to top](#readme-top))</p>
+[back to top](#readme-top)
 
 ## 🤝 Contributing <a name="contributing"></a>
 
 Contributions, issues, and feature requests are welcome!
 
 Feel free to check the [issues page](https://github.com/paulinagonzalezc/blog-app-rails/issues).
 
-<p align="right">([back to top](#readme-top))</p>
+[back to top](#readme-top)
 
 ## ⭐️ Show your support <a name="support"></a>
 
 If you like this project please give it a star 😁🌟✨
 
-<p align="right">([back to top](#readme-top))</p>
+[back to top](#readme-top)
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
 Microverse.
+[back to top](#readme-top)
 
-<p align="right">([back to top](#readme-top))</p>
-
-## 📝 License <a name="license"></a>
+## 📝 License <a name="license" href="https://github.com/Mov305/To-do-list/blob/master/MIT.md" ></a>
 
 This project is [MIT](./LICENSE) licensed.
```

Comment:
Great job so far @Mov305 

- [ ] Please fix the link to the MIT license it's broken.
You can follow the [README Template](https://github.com/CollinsTatang/readme-template)

Reply:
Got it thanks